spacialgaze-plugins/SG.js

```diff
@@ -92,7 +92,7 @@ SG.reloadCSS = function () {
 		path: '/customcss.php?server=' + (Config.serverid || cssPath),
 		method: 'GET',
 	};
-	http.get(options);
+	http.get(options, () => Monitor.adminlog("Successfully reloaded the server CSS."));
```

Comment:
iirc this function runs on a timer, so this would spam the upper staff room.

Reply:
And console?

Reply:
not the console iirc

Reply:
It would spam console as well yes